### DISCUSSION TITLE
Is there any othere animations than zoom itself?

### DISCUSSION THREAD

**Discussion by pineapples5972** [yshui/picom#841] [Q&A]
I couldnt found more animation method other than zoom. Are there any or its just zoom?

**Comment by beyond9thousand** [comment]
> I couldnt found more animation method other than zoom. Are there any or its just zoom?

what fork are you using?

**Reply by pineapples5972** [reply]
Nvm I found in config zoom,  fly-in,  slide-down, slide-up, slide-right, slide-left 
I am using this fork: https://github.com/dccsillag/picom/tree/implement-window-animations
